src/collections/_documentation/development/server/environment.md

```diff
@@ -155,8 +127,19 @@ And after doing that, reload your shell:
 PATH="" exec /bin/bash -l
 ```
 
-Now, direnv should automatically execute (or you might need to type `direnv allow`). If you've followed this guide correctly, it should succeed.
+Any time the `.envrc` configuration changes (including the first time) you will be prompted to run `direnv allow` before any of the configuration will run.
+
+## Bootstrap Services
+
+The last step is to run `make bootstrap`. This will take a long time, as it bootstraps not only Sentry and its dependencies, but starts up related services, and runs database migrations.
 
+The `bootstrap` command does a few things you'll want to know about:
+
+- `sentry init` creates the baseline Sentry configuration in `~/.sentry/`.
+- `sentry devservices up` spins up required Docker services (such as Postgres and Clickhouse)
+- `sentry upgrade` runs Postgres migrations, and will also prompt you to create a user. You will want to ensure your first user is designated as **superuser**.
```

Comment:
Idea: keep an up-to-date postgres image/volume so people can just download it instead of running all the migrations from scratch?

Reply:
maybe useful for someone whos not _really_ developing on Sentry, but not really useful for staff since migrations change frequently

we'd be better off dumping to .sql and just letting them load that, but then we're just duplicating the migration process